src/subscriptions/restHookLambda/restHook.ts

```diff
@@ -67,8 +66,9 @@ export default class RestHookHandler {
         event: SQSEvent,
         allowListPromise: Promise<{ [key: string]: AllowListInfo }>,
     ): Promise<SQSBatchResponse> {
-        await ensureAsyncInit(allowListPromise);
+        console.log(allowListPromise);
```

Comment:
Use `logger.debug` instead? The allowList might contain sensitive information that customers do not want in the log all the time.